src/PrestaShopBundle/Resources/views/Admin/Common/Grid/Blocks/Table/filters_row.html.twig

```diff
@@ -42,4 +42,8 @@
       {% endif %}
     </td>
   {% endfor %}
+
+  {% for filter in grid.hidden_column_filters %}
```

Comment:
I'm not sure it's really necessary Since the hidden filter is added in the columns already and is associated with the category column shouldn't it have been displayed already? It just means two form fields are displayed in the column no?